common/config/config.go

```diff
@@ -515,6 +519,23 @@ type (
 		// URI is the domain default URI for visibility archiver
 		URI string `yaml:"URI"`
 	}
+
+	// Partitioner is the configuration for the partitioning library, a library
+	// for segmenting portions of workflows into isolation-groups - a resiliency
+	// concept meant to help move workflows around and away from failure zones.
+	Partitioner struct {
+		IsolationGroupEnabled dynamicconfig.BoolPropertyFnWithDomainFilter
```

Comment:
dynamic configs don't belong to this package
if you search TODO, we're moving them out of this package

Reply:
I think we can remove them. They don't seem to be necessary.
Just use the dynamic config by names directly.